packages/server-wallet/src/protocols/defunder.ts

```diff
@@ -77,6 +79,8 @@ export class Defunder {
       await ChainServiceRequest.insertOrUpdate(channel.channelId, 'pushOutcome', tx);
       await this.chainService.pushOutcomeAndWithdraw(
         adjudicatorStatus.states[0],
+        // todo: we are assuming that we submitted the challenge.
```

Comment:
It's nice to be consistent. `TODO` is highlighted by vs-code (or at least by a common plugin?). `todo` is not.

```suggestion
        // TODO: we are assuming that we submitted the challenge.
```

Reply:
Consistency sounds good to me! `todo` is not highlighted in my editor regardless of case.